autopark/PrintVehicle.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace autopark
+{
+    static class PrintVehicle
+    {
+        public static string ToString(Vehicle obj)
+        {
+            return  $"{obj.VehicleType} {obj.ModelName} " + 
+                    $"{obj.RegistrationNumber} {obj.WeightKg} " + 
+                    $"{obj.ManufactureYear} {obj.MileageKm} " + 
+                    $"{obj.Color} {obj.GetCalcTaxPerMonth().ToString("0.00")}";
+        }
```

Comment:
Unnecessary class. Override ToString method of Vehicle class.

Reply:
![image](https://user-images.githubusercontent.com/79978383/115914339-7d4b8a00-a47a-11eb-8353-1794e3dd9297.png)
![image](https://user-images.githubusercontent.com/79978383/115914452-9fdda300-a47a-11eb-9741-9e523a93f233.png)
Done